docs/BottomNavigation.md

```diff
@@ -7,7 +7,7 @@
 
 ```js
 ...
-import { BottomNavigation } from 'react-native-material-ui';
+import { BottomNavigation } from '../react-native-material-ui';
```

Comment:
It should be just `react-native-material-ui`. Why do you think it should be relative path?

Reply:
I think it should just be `react-native-material-ui`. I was just following the other docs

Reply:
I will update :)

Reply:
Updated. and made all the others the same.